_targets.R

```diff
@@ -32,31 +34,35 @@ if (!have_gghdx) {
 
 
 library(targets)
-library(tidyverse)
-library(sf)
-library(terra)
-library(exactextractr)
-library(lubridate)
-library(readxl)
-library(janitor)
-library(zoo)
-library(rgee)
-library(ggrepel)
-library(tidyrgee)
-library(gghdx)
+# library(tidyverse)
```

Comment:
I switched to loading the packages in the `_targets.R ` file using tar_option_set()` functionality rather than using `library()`...when targets are created `tar_make()` the environment is cleared. Therefore, I had lazily left them in so I could quickly uncomment and access the libraries for debugging and developement.

I think what might make more sense is just to create a utils function `load_libs()` which will load them all for debugging/development purposes and remove the commented out libraries

Reply:
removed the commented out lib calls and implemented the above here: 57cca357d5943dcccd60614cff0262013fccdf24